Makefile

```diff
@@ -42,11 +45,10 @@ test: $(TESTS)
 
 # run specific test
 .PHONY: $(TESTS)
-$(TESTS): .test-image $(CACHE_DIR)/updated
+$(TESTS): .test-image $(CACHE_DIR)/updated keys/testuser
 	./run-in-docker.sh $@
 
 
-# create a backend and drop you into a shell on it
```

Comment:
is this intentional removal of comment, seems unrelated?

Reply:
yeah, it was dangling bad comment